ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -4384,6 +4384,76 @@ tests:
   steps:
     cluster_profile: nutanix-qe-dis
     workflow: agent-qe-nutanix-disconnected
+- as: aws-short-cert-rotation-f9
+  cron: 42 11 2,11,23 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - chain: openshift-e2e-test-qe-cert-rotation
+    workflow: openshift-e2e-cert-rotation-short
```

Comment:
How about use similar install workflow - cucushift-installer-rehearse-aws-ipi like other platforms.

Reply:
@jianlinliu That workflow created only for AWS by dev we can't use that.

Reply:
`workflow: openshift-e2e-cert-rotation-short` is defined for aws-ipi  which is the original CI test, others we did refer to it.